packages/launchpad/src/setup/EnvironmentSetup.vue

```diff
@@ -74,6 +73,7 @@ fragment EnvironmentSetup on Wizard {
     isDetected
     type
     category
+    codeGenFramework
```

Comment:
Now that we are sorting alphabetically, this is unused. Looks like we can revert the changes to the `CodeGenFrameworkEnum` and `FrameworkCategoryEnum`

Reply:
good piont, done